### PR TITLE
[fuzzing] coverage needs to run with fuzzing flag enabled

### PR DESCRIPTION
**Pull request by mimoo**
just updating the README

### CODE REVIEW

File: testsuite/libra-fuzzer/README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ cargo run --bin investigate -- -i artifacts/compiled_module/crash-5d7f403f
 The following command will run libfuzzer on the relevant target with your input:
 
 ```
-// build single fuzzer for target using instruction in the 'google oss-fuzz integration' section
+# build single fuzzer for target using instruction in the 'google oss-fuzz integration' section
```
Comment (davidiw) [inline testsuite/libra-fuzzer/README.md:74]: why the change?

File: testsuite/libra-fuzzer/README.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ See [Google OSS-Fuzz's documentation on reproducing bugs](https://google.github.
 To test coverage of our fuzzers you can run the following command with [grcov](https://github.com/mozilla/grcov):
 
 ```
-CORPUS_PATH=fuzz/corpus cargo xtest --html-cov-dir <some path for html output> -p libra-fuzzer -- coverage
+RUSTFLAGS='--cfg feature="fuzzing"' CORPUS_PATH=fuzz/corpus cargo xtest --html-cov-dir <some path for html output> -p libra-fuzzer -- coverage
```
Comment (davidiw) [inline testsuite/libra-fuzzer/README.md:89]: we should code mod this to testing since it is used for both purposes.

Comment (rexhoffman) [reply testsuite/libra-fuzzer/README.md:89]: Pretty sure that xtest, which turns on all features makes this unnecessary.   Also confident that x will ignore this set of flags and not pass them through.   Confident that this is a no op.

File: testsuite/libra-fuzzer/README.md (hunk 1 above)
Comment (mimoo) [reply testsuite/libra-fuzzer/README.md:74]: that's a bash comment
